packages/shared/src/index.ts

```diff
@@ -48,8 +48,12 @@ export const objectToString = Object.prototype.toString
 export const toTypeString = (value: unknown): string =>
   objectToString.call(value)
 
+export function toRawType(value: unknown): string {
+  return toTypeString(value).slice(8, -1)
+}
+
 export const isPlainObject = (val: unknown): val is object =>
-  toTypeString(val) === '[object Object]'
+  toRawType(val) === 'Object'
```

Comment:
I would revert this line since it avoids an extra slice call.

Reply:
thx~